### PR TITLE
[1.16] fix #262 #263 #264 and 2 more things (read comments)

### LINKED ISSUE #262: Graphical Issue When Sieving Blocks
**Describe the bug**
When Sieving blocks (dust, gravel, sand, etc.) The texture of the block shows through the side of the sieve. This seems to only happen on the South and East side of the sieve blocks. See picture below

**To Reproduce**
1. Place down sieve
2. sieve a dust block
3. observe the graphical issue

**Expected behavior**
The block being sieved should not show through the side of the sieve

**Version**
Minecraft Version: 1.16.5
Forge Version: 36.2.4
Ex Nihilo: Sequentia Version: 20210826035238

**Additional context**
image

[AB#82](https://dev.azure.com/novamachina/9b4371f9-393b-4d7d-8cb9-1f6c864e3511/_workitems/edit/82)

### PR DESCRIPTION
**Pull request by remplerus**

### CODE REVIEW

**Comment by remplerus** [comment]
removed temp added makro

**Comment by remplerus** [comment]
fix #262 #264 and adds dropping item inserted into sieve when breaking

**Comment by remplerus** [comment]
fix #263 and jei not showing fluids correct in crucible tab when using crafttweaker

**Comment by remplerus** [comment]
fixes #265

File: src/main/java/novamachina/exnihilosequentia/common/utility/Config.java [hunk 1]
```diff
@@ -337,6 +325,8 @@ private static void crucibleConfigs() {
     private static void debugConfigs() {
         enableDebugLogging = COMMON_BUILDER.comment("Enable extra logging? (Default: false)")
                 .define("enableDebugLogging", false);
+        enableStoneblock = COMMON_BUILDER.comment("Enable Stoneblock settings? (Default: false)")
+                .define("enableStoneblock", false);
```
Comment (NovaMachina) [inline src/main/java/novamachina/exnihilosequentia/common/utility/Config.java]: This is a little vague for what the config does. Can we get some more info here?

**Review by NovaMachina — CHANGES_REQUESTED** [review]
Everything looks great except for the description for the enableStoneblock config. Once that has a better description, this is good to go.

File: src/main/java/novamachina/exnihilosequentia/common/utility/Config.java (hunk 1 above)
Comment (remplerus) [reply src/main/java/novamachina/exnihilosequentia/common/utility/Config.java]: will be removed in other pr. moving to own mod

File: src/main/java/novamachina/exnihilosequentia/common/utility/Config.java [hunk 2]
```diff
@@ -450,6 +435,6 @@ private static void sieveConfigs() {
                 .defineInRange("meshStackSize", 64, 1, 64);
         maxSieveClicks = COMMON_BUILDER
                 .comment("The number of sieve clicks required to sieve a block. (Default: 10)")
-                .defineInRange("maxSieveClicks", 10, 1, 10);
+                .defineInRange("maxSieveClicks", 10, 1, 20);
```
Comment (NovaMachina) [inline src/main/java/novamachina/exnihilosequentia/common/utility/Config.java]: Why did this get increased?

Comment (remplerus) [reply src/main/java/novamachina/exnihilosequentia/common/utility/Config.java]: idk what happened there...

**Review by NovaMachina — APPROVED** [review]

**Pull request merged**